CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 ## Latest
 * Update Razor to 7.0.0-preview.23616.1 (PR: [#6760](https://github.com/dotnet/vscode-csharp/pull/6760)
 * Update Roslyn to 4.9.0-3.23611.3 (PR: [#6736](https://github.com/dotnet/vscode-csharp/pull/6736))
+  * Fixed issue where Razor trace level was ignored. New `razor.server.trace` setting created that is not ignored (PR: [dotnet/razor#9745](https://github.com/dotnet/razor/pull/9745))
```

Comment:
Should this be under `* Update Razor to...`?

```suggestion
* Update Razor to 7.0.0-preview.23616.1 (PR: [#6760](https://github.com/dotnet/vscode-csharp/pull/6760)
  * Fixed issue where Razor trace level was ignored. New `razor.server.trace` setting created that is not ignored (PR: [dotnet/razor#9745](https://github.com/dotnet/razor/pull/9745))
* Update Roslyn to 4.9.0-3.23611.3 (PR: [#6736](https://github.com/dotnet/vscode-csharp/pull/6736))
```

Reply:
Whoops, yep!

Needs updating anyway to whatever the new build of Razor is that has the right change. Thanks for the catch though :)